tasks/emacs.py

```diff
@@ -0,0 +1,43 @@
+"""
+Emacs namespaced tags
+
+Helpers for getting Emacs set up nicely
+"""
+from invoke import task
+
+from .build_tags import build_tags, filter_incompatible_tags, get_build_tags, get_default_build_tags
+from .flavor import AgentFlavor
+
+
+@task
+def set_buildtags(
+    _,
+    target="agent",
+    build_include=None,
+    build_exclude=None,
+    flavor=AgentFlavor.base.name,
+    arch="x64",
+):
+    """
+    Create Emacs .dir-locals.el settings file for this project to include correct build tags
+    """
+    flavor = AgentFlavor[flavor]
+
+    if target not in build_tags[flavor].keys():
+        print("Must choose a valid target.  Valid targets are: \n")
+        print(f'{", ".join(build_tags[flavor].keys())} \n')
+        return
+
+    build_include = (
+        get_default_build_tags(build=target, arch=arch, flavor=flavor)
+        if build_include is None
+        else filter_incompatible_tags(build_include.split(","), arch=arch)
+    )
+    build_exclude = [] if build_exclude is None else build_exclude.split(",")
```

Comment:
you could maybe define default value of `build_exclude` to `""` to save this `if` condition